package.json

```
@@ -49,5 +52,9 @@
 	"bugs": {
 		"url": "https://github.com/iSirThijs/jungle-gym/issues"
 	},
-	"type": "module"
+	"type": "module",
+	"lint-staged": {
+		"*.js": "eslint --cache --fix",
+		"*.{js,css,ts,md,svelte,json}": "prettier --write"
```

Misschien nog `--plugin-search-dir` toevoegen
```suggestion
		"*.{js,css,ts,md,svelte,json}": "prettier --plugin-search-dir=. --write"
```